Learn-Information-Security-by-Building-a-Contact-Form/server080.js

```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+const app = express();
+const helmet = require("helmet");
+// https://helmetjs.github.io/ - for more information on all packages and uses.
+
+app.use(helmet());
+
+// You can enable additional security features, disable a feature, or change the default settings.
```

Comment:
This could be broken up into two steps, first prompting the learner to create the `helmetConfig` object, then passing it to `app.use(helmet())`.